code/datums/components/butchering.dm

```diff
@@ -98,6 +98,7 @@
 								"<span class='notice'>You butcher [meat].</span>")
 	ButcherEffects(meat)
 	meat.harvest(butcher)
+	meat.log_message("has been butchered by [key_name(butcher)]", LOG_ATTACK)
```

Comment:
This should log to both attack and deaths

Reply:
added in 0429d27.

INVESTIGATE_DEATHS already filters out non-ckeyed mobs, so theres no need to check for ckeys before logging